python/src/trezorlib/cli/trezorctl.py

```diff
@@ -267,6 +275,22 @@ def change_pin(connect, remove):
     return device.change_pin(connect(), remove)
 
 
+@cli.command()
+@click.argument("operation", type=CHOICE_SD_SALT_OPERATION_TYPE)
+@click.pass_obj
+def sd_salt(connect, operation):
+    """Configure SD card salt protection.
+
+    Enable, disable or regenerate SD card salt. The options are:
+
+    \b
```

Comment:
What does `\b` mean in this context?

Reply:
By default Click rewraps text based on the current width of the terminal. \b disables this behavior, i.e. it forces Click to preserve newlines.

Reply:
TIL